libs/core/langchain_core/messages/modifier.py

```diff
@@ -0,0 +1,38 @@
+from typing import Any, List, Literal
+
+from langchain_core.messages.base import BaseMessage, BaseMessageChunk
+
+
+class ModifierMessage(BaseMessage):
+    """Message responsible for modifying other messages (deleting / updating.)"""
+
+    def __init__(self, id: str, **kwargs: Any) -> None:
+        return super().__init__("modifier", id=id)
```

Comment:
yea, can just be an empty string, wasn't sure which is less confusing `""`

Reply:
If we don't need the content, do we want to change the hierarchy so that content is only for messages that contain content? Or could the content be helpful here?